src/class-elements/grammar-special-prototype-accessor-meth-valid.case

```diff
@@ -0,0 +1,22 @@
+// Copyright (C) 2020 Alexey Shvayka. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: Accessor Methods can be named "prototype"
+info: |
+    Class Definitions / Static Semantics: Early Errors
+
+    ClassElement : MethodDefinition
+        It is a Syntax Error if PropName of MethodDefinition is not "constructor" and HasDirectSuper of MethodDefinition is true.
+        It is a Syntax Error if PropName of MethodDefinition is "constructor" and SpecialMethod of MethodDefinition is true.
+template: syntax/valid
+---*/
+
+//- elements
+get prototype() { return 13; }
+set prototype(_) {}
+//- teardown
+assert(C.hasOwnProperty('prototype'));
+assert(C.prototype.hasOwnProperty('prototype'));
+assert.notSameValue(C.prototype.prototype, C.prototype);
```

Comment:
this should be extended to the property description, as it differentiates comparing `C.prototype` from `C.prototype.prototype`.

```js
// having included propertyHelper.js above...

verifyProperty(C.prototype, 'prototype', {
  // writable: true, // for the non accessor cases
  configurable: true,
  enumerable: false,
});
```

I wonder if we should also observe the accessor methods here, it's a bit beyond the syntax as the template intents to, but it might be fine. In this case, it should be tested before `verifyProperty`.